src/modules/functions.js

```diff
@@ -1,4 +1,4 @@
-let myTask = JSON.parse(localStorage.getItem('myTask')) || [];
+let myTask = JSON.parse(localStorage.getItem('myTask')) || []; // eslint-disable-line
```

Comment:
- I would advise you not to disable the linter. The[ linter](https://github.com/microverseinc/linters-config) is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Can you solve it manually?